src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritHandler.java

```diff
@@ -577,4 +641,37 @@ public void shutdown(boolean join) {
     public void triggerEvent(GerritEvent event) {
         post(event);
     }
+    /**
+     * schedules update of list
+     */
+    private void scheduleGerritWhitelistRead() {
+        final Runnable whitelistReader = new Runnable() {
```

Comment:
```suggestion
        if (StringUtils.isEmpty(System.getProperty(whitelistLocation))) {
            return;
        }
        final Runnable whitelistReader = new Runnable() {
```
Otherwise there will be error messages in the log even though the user doesn't want this feature.